spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/reload/ConfigReloadAutoConfigurationTest.java

```diff
@@ -55,64 +64,78 @@ public static void setUpBeforeClass() {
 
 		HashMap<String, String> data = new HashMap<>();
 		data.put("bean.greeting", "Hello ConfigMap, %s!");
-		server.expect().withPath("/api/v1/namespaces/test/configmaps/" + APPLICATION_NAME)
```

Comment:
mock client works slightly differently now and there is no need to mock paths, instead everything happens on the client itself. It is the same exact thing, logically.